src/client/pmix_client.c

```diff
@@ -964,8 +961,14 @@ static void _commitfn(int sd, short args, void *cbdata)
      * that we contributed whatever we had */
     PMIX_PTL_SEND_RECV(rc, &pmix_client_globals.myserver, msgout,
                        wait_cbfunc, (void*)&cb->active);
+    if (PMIX_SUCCESS == rc) {
```

Comment:
@rhc54
I'd appreciate if when possible you'd reopen Boris PR, merge it and do the refactoring.